### DISCUSSION TITLE
Android Build Environment

### DISCUSSION THREAD

**Discussion by adepaoli** [strongswan/strongswan#690] [General]
Are you so kindly telling me in which environment you are building everything, please?

- Linux distro and version
- Android Studio Version
    - Android SDK Version
    - Android NDK Version
I think this informations could be useful to others and should be in the Wiki Guide on how to build Android.
I'm using Windows 10 with the latest Android Studio and Ubuntu 20  (WSL2) but I'm running in multiple compile errors.

Thanks
Andrea

**Comment by tobiasbrunner** [comment]
Shouldn't really matter as long as you don't use Windows :) (Not sure if WSL helps at all.)

**Comment by adepaoli** [comment]
Thank you for your quick reply.
Which Linux distro are you suggesting?

**Reply by tobiasbrunner** [reply]
Any should work, so it really comes down to your personal preferences. Just found [distrochooser.de](https://distrochooser.de/), maybe that could be helpful to you :)

**Comment by adepaoli** [comment]
Thanks, one more question. Can I use Mac Os? 
I would like to avoid using a virtual machine to install Linux distro + android studio ...

**Reply by tobiasbrunner** [reply]
Possible, but I don't know, sorry.

**Comment by realbitt** [comment]
this project hates windows so badly ;)
My journey to success :
Os : ubunto 20.04 with GUI
android studio : Arctic fox 2020.3.1 Patch 3
NDK Version : 20.1.5948944 (very important)

good luck
